src/main/java/org/spongepowered/api/plugin/PluginManager.java

```diff
@@ -76,4 +90,53 @@
      */
     boolean isLoaded(String id);
 
+    /**
+     * Gets the id of the {@link Plugin}.
+     *
+     * @return The id
+     */
+    default String getId(Object plugin) {
```

Comment:
I don't think this is a good idea, especially because #1081 adds much more things here. Why not call `find(plugin).getId()` directly? I'd doubt this would make a big difference.